### ISSUE TITLE
Run moon rider , There is a possibility that the image is frozen when the block wall appears(with a high probability).

### ISSUE THREAD

**Issue by dingsing2021** [Igalia/wolvic#35]
## Configuration

Wolvic version: 0.9.3
Wolvic build ID: 

Hardware: huawei vr glass 6dof + mate40pro(kirin 9000 chipset)

## Steps to Reproduce

1.open wolvic and enter moon rider
2. chose the classic mode, chose the song: beat it - Michael Jackson
3. When the block wall appears, the picture freezes (with a high probability), but the sound is normal. The 6dof controller buttons can be used.

## Current Behavior

1. chose the classic mode, chose the song: beat it - Michael Jackson
2. When the block wall appears, the picture freezes (with a high probability), but the sound is normal. The 6dof controller buttons can be used.
## Context

You can confirm that the phone is not overheating.

## Error Logs and Stack Traces

<details open>

```
```

</details>

**Comment by svillar** [comment]
I've tried hard to reproduce this with no luck. Perhaps you're testing with some experimental runtime?

**Comment by dingsing2021** [comment]
> I've tried hard to reproduce this with no luck. Perhaps you're testing with some experimental runtime?
the huawei VR runtime version is  3.5.0.75;  
Could it be related to the network speed? Because I'm in Chinese mainland and moon rider's server is overseas?
I will continue to try to find out how this happens.
I'd like to test it on version 3.5.0.76 runtime, but this wolvic VR Browser has not been released yet,and I hope it will be released as soon as possible.

**Comment by dingsing2021** [comment]
> 

Uploading VID_20220303113023.mp4…

here is the screen record ing.
it is very easy to reproduction in my environment.

**Comment by dingsing2021** [comment]
https://user-images.githubusercontent.com/86649949/156492733-5c9b07c4-8c96-4f68-b619-23698c8284b9.mp4

here is the record video. you can see when it frozen.
it seems very easy to happen in my environment.

**Comment by svillar** [comment]
OK I can reproduce it now. The key here is that you have yo be in "Classic Mode". If "Punch Mode" is selected for example then the issue is not reproducible for me.

**Comment by dingsing2021** [comment]
> 

yes. you are right.
the steps to reproduce I and the video posted mentioned the Classic mode.

**Comment by svillar** [comment]
So this is pretty interesting I found the following. Using the same song if you play it a bit using "Punch mode" and then go back to menu and then select "Classic Mode" and then play it then it does not hang. The application must be doing something weird... I'll keep debugging the issue

**Comment by svillar** [comment]
OK so I've just found the actual problem behind this issue after seeing this in the logs.
```
2022-08-11 20:00:26.070 32714-3944/com.igalia.wolvic.dev E/Web Content: [JavaScript Error: "TypeError: can't access property "pulse", this.gamepad.hapticActuators[n.actuatorIndex] is undefined" {file: "https://moonrider.xyz/build/build.js?v=3" line: 342}]
```

When you hit the wall, the application triggers some haptic feedback on the controllers. Haptic feedback is enabled in Oculus backend but not in OpenXR. We're now working on implementing it as part of issue #222. Once this is implemented this issue would be gone.

**Comment by svillar** [comment]
This is fixed now in main after landing #222. It'll work properly in the upcoming 1.1 release.
